AUTHORS.md

```diff
@@ -24,3 +24,4 @@ The following people have contributed code and/or documentation to the project
 * [William Schlitzer](https://github.com/willschlitzer) | [0000-0002-5843-2282](https://orcid.org/0000-0002-5843-2282) | Unaffiliated
 * [Yohai Magen](https://github.com/yohaimagen) | [0000-0002-4892-4013](https://orcid.org/0000-0002-4892-4013) | Tel Aviv University, Israel
 * [Yvonne Fröhlich](https://github.com/yvonnefroehlich) | [0000-0002-8566-0619](https://orcid.org/0000-0002-8566-0619) | Karlsruhe Institute of Technology, Germany
+* [Jing-Hui Tong](https://github.com/JingHuiTong) | [0009-0002-7195-3071](https://orcid.org/0009-0002-7195-3071) | National Taiwan Normal University, Taiwan
```

Comment:
Could you please move this to line 16 in this file? We'd like to keep the list in alphabetical order.

Reply:
Hi Wei Ji, I would like to be added to @GenericMappingTools/pygmt-contributors!

Reply:
Actually, let me check with the others yet. @GenericMappingTools/pygmt-maintainers, ok with adding @JingHuiTong to the contributors team?

Reply:
Sounds good to me.

Reply:
Hmm, I can't seem to add people to the team... Maybe because I'm not a GMT org admin? @seisman or @maxrjones, could you trying adding her please?

Reply:
Welcome @JingHuiTong :tada:. Great to see the contributors team growing :rocket:.